### DISCUSSION TITLE
Can't Remove new Win11 Tab Control Interface

### DISCUSSION THREAD

**Discussion by eQuilter** [valinet/ExplorerPatcher#1402] [General]
I have installed ExplorerPatcher 22621.1555.55.1, and I'm running Windows 11 OS Build 22621.1555, and I can't remove the tabs in explorer. I've tried both Windows 10 Ribbon and Windows 7 Command Bar settings, and restarted file explorer, but the new Windows 11 Tabs are still there. The worst part about the new tabs is that there's no line to enter a path to a folder or network address.
What can be done?

**Comment by eQuilter** [comment]
For reference, here is a screenshot of my explorer window that is MISSING the address bar:

![Windows11MissingSearchBar](https://user-images.githubusercontent.com/132214115/235687187-7a522581-e976-4935-8400-c1599fea25fc.jpg)

**Reply by vippium** [reply]
Try to disable the "Disable Navigation Bar" option in File Explorer in EP Settings

**Reply by eQuilter** [reply]
I'm not finding any setting for "Disable Navigation Bar" in File Explorer.
Where is it???

**Reply by vippium** [reply]
Go to EP properties by r-clicking on taskbar and selecting properties.

Here after, select File Explorer option and disable the option.

**Reply by eQuilter** [reply]
Gosh, that was it, as "Disable navigation Bar" was checked.
I unchecked it, and now the navigation bar has returned.
For reference, here is the setting in question:

![NavigationBar](https://user-images.githubusercontent.com/132214115/235712605-9a6ed728-5960-4cd9-a5f3-8886250d2510.jpg)

**Reply by vippium** [reply]
Well...No Problem....why don't you use the "register Shell extension" option as it is good for Open/Save dialog boxes. It allows these explorer settings to work in those dialog also.

**Reply by eQuilter** [reply]
Good idea; I've set the 'Register Shell Extension".
